From 3dd9af3221d2a4ea4caf2865bac5fe9aaf2e2643 Mon Sep 17 00:00:00 2001
From: YouMin Chen <cym@rock-chips.com>
Date: Thu, 17 May 2018 19:18:34 +0800
Subject: [PATCH] arm64: dts: rockchip: px30: modify dram default timing

Set the default value for pd_idle,sr_idle and standby_idle.

Change-Id: Idfa2d62229fd8ec1ab48c2506a6b5ba3da080a49
Signed-off-by: YouMin Chen <cym@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi b/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
index b46414ff1335..f429cc8669ab 100644
--- a/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30-dram-default-timing.dtsi
@@ -13,11 +13,11 @@
 		ddr2_speed_bin = <DDR2_DEFAULT>;
 		ddr3_speed_bin = <DDR3_DEFAULT>;
 		ddr4_speed_bin = <DDR4_DEFAULT>;
-		pd_idle = <0>;
-		sr_idle = <0>;
+		pd_idle = <13>;
+		sr_idle = <93>;
 		sr_mc_gate_idle = <0>;
 		srpd_lite_idle	= <0>;
-		standby_idle = <0>;
+		standby_idle = <4000>;
 
 		auto_pd_dis_freq = <1066>;
 		auto_sr_dis_freq = <800>;
-- 
2.35.3

